stats/lib/platform-routes.js

```diff
@@ -3,6 +3,7 @@ import {
   fetchDailyStationCount,
   fetchMonthlyStationCount,
   fetchDailyRewardTransfers,
+  fetchTopMeasurementParticipants,
```

Comment:
topMeasurementParticipants is a bit awkward, what about topMeasurementsParticipants?

Reply:
or fetchParticipantsWithTopMeasurements

Reply:
Sure, addressed in https://github.com/filecoin-station/spark-stats/pull/164/commits/cf96533b9c84f7a2e4b9cfe2dc871bd6be1c1831